specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/examples/VpnServerConfigurationGet.json

```diff
@@ -0,0 +1,69 @@
+{
+  "parameters": {
+    "vpnServerConfigurationName": "vpnServerConfiguration1",
+    "resourceGroupName": "rg1",
+    "api-version": "2019-06-01",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/vpnServerConfigurations/vpnServerConfiguration1",
+        "etag": "w/\\00000000-0000-0000-0000-000000000000\\",
+        "location": "West US",
+        "name": "vpnServerConfiguration1",
+        "type": "Microsoft.Network/vpnServerConfigurations",
+        "tags": {
+          "key1": "value1"
+        },
+        "properties": {
+            "provisioningState": "Succeeded",
+			"vpnProtocols": [
+			  "IkeV2"
+			],
+			"vpnAuthenticationTypes": [
+			  "Certificate"
+			],		
+			"vpnClientIpsecPolicies": [
+			  {
+				"saLifeTimeSeconds": 86472,
+				"saDataSizeKilobytes": 429497,
+				"ipsecEncryption": "AES256",
+				"ipsecIntegrity": "SHA256",
+				"ikeEncryption": "AES256",
+				"ikeIntegrity": "SHA384",
+				"dhGroup": "DHGroup14",
+				"pfsGroup": "PFS14"
+			  }
+			],
+			"vpnServerConfigVpnClientRootCertificates": [
+			  {
+				  "name": "vpnServerConfigVpnClientRootCert1",				
+				  "publicCertData": "MIIC5zCCAc+gAwIBAgIQErQ0Hk4aDJxIA+Q5RagB+jANBgkqhkiG9w0BAQsFADAWMRQwEgYDVQQDDAtQMlNSb290Q2VydDAeFw0xNzEyMTQyMTA3MzhaFw0xODEyMTQyMTI3MzhaMBYxFDASBgNVBAMMC1AyU1Jvb3RDZXJ0MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArP7/NQXmW7cQ/ZR1mv3Y3I29Lt7HTOqzo/1KUOoVH3NItbQIRAQbwKy3UWrOFz4eGNX2GWtNRMdCyWsKeqy9Ltsdfcm1IbKXkl84DFeU/ZacXu4Dl3xX3gV5du4TLZjEowJELyur11Ea2YcjPRQ/FzAF9/hGuboS1HZQEPLx4FdUs9OxCYOtc0MxBCwLfVTTRqarb0Ne+arNYd4kCzIhAke1nOyKAJBda5ZL+VHy3S5S8qGlD46jm8HXugmAkUygS4oIIXOmj/1O9sNAi3LN60zufSzCmP8Rm/iUGX+DHAGGiXxwZOKQLEDaZXKqoHjMPP0XudmSWwOIbyeQVrLhkwIDAQABozEwLzAOBgNVHQ8BAf8EBAMCAgQwHQYDVR0OBBYEFEfeNU2trYxNLF9ONmuJUsT13pKDMA0GCSqGSIb3DQEBCwUAA4IBAQBmM6RJzsGGipxyMhimHKN2xlkejhVsgBoTAhOU0llW9aUSwINJ9zFUGgI8IzUFy1VG776fchHp0LMRmPSIUYk5btEPxbsrPtumPuMH8EQGrS+Rt4pD+78c8H1fEPkq5CmDl/PKu4JoFGv+aFcE+Od0hlILstIF10Qysf++QXDolKfzJa/56bgMeYKFiju73loiRM57ns8ddXpfLl792UVpRkFU62LNns6Y1LKTwapmUF4IvIuAIzd6LZNOQng64LAKXtKnViJ1JQiXwf4CEzhgvAti3/ejpb3U90hsrUcyZi6wBv9bZLcAJRWpz61JNYliM1d1grSwQDKGXNQE4xuN"
+			  }
+			],
+			"vpnServerConfigVpnClientRevokedCertificates": [
+			  {
+				  "name": "vpnServerConfigVpnClientRevokedCert1",								  
+				  "Thumbprint": "83FFBFC8848B5A5836C94D0112367E16148A286F"
```

Comment:
Thumbprint -> thumbprint.
Bunch of other hits for the same in the  other examples

Reply:
Just fixed the same.